solutions/account-infrastructure-base/variables.tf

```diff
@@ -27,13 +27,13 @@ variable "region" {
 variable "prefix" {
   type        = string
   description = "An optional prefix to append to all resources created by this solution. If `provision_atracker_cos` is true, this value will be converted to lowercase in all instances."
```

Comment:
Please include the details about prefix being `"" (empty string)`, or `null` for advanced users.